scripts/hosts-add.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# Add new IP-host pair to /etc/hosts.
+
+if [[ "$1" && "$2" ]]
+then
+    IP=$1
+    HOSTNAME=$2
+
+    if [ -n "$(grep $HOSTNAME /etc/hosts)" ]
+        then
+            echo "$HOSTNAME already exists:";
+            echo $(grep $HOSTNAME /etc/hosts);
+        else
+            sudo sed -i "/#### HOMESTEAD-SITES-BEGIN/c\#### HOMESTEAD-SITES-BEGIN\\n$IP\t$HOSTNAME\twww.$HOSTNAME" /etc/hosts
+
+            if ! [ -n "$(grep $HOSTNAME /etc/hosts)" ]
```

Comment:
why would it fail to add? if the markers don't exist?

Reply:
I suppose it could fail for any number of reasons, but the check is basically saying, "If the markers are not found, complain about it".

FWIW, I grabbed that, verbatim, from https://github.com/laravel/homestead/pull/1007#issuecomment-489484013 and didn't clean it up beyond reducing verbosity for the success cases. We can certainly eliminate any error-handling in this use-case; it looks like the code was originally written to be a bit more modular and used within a function.

Reply:
Upon further consideration, I think it makes sense to leave the output as-is for failure scenarios, only because it will make problems more obvious; it seems less than ideal to ignore failures silently.